850.split-ambiguities/g.yaml

```diff
@@ -1,5 +1,8 @@
 # vim: tabstop=29 expandtab softtabstop=29 nomodeline
 
+- { name: g, wwwpart: g.equationzhao, setname: g-ls }
```

Comment:
There needs to be a rule for another counterpart, e.g. `wwwpart: knu, setname: g-grep-wrapper`